data.csv

```diff
@@ -0,0 +1,22 @@
+Proto_list,Blacklist,Banwords
+#netsec,Pornhub,porn
+#DNS,xnxx,pornhub
+#DANE,,xvideo
+#DNSSEC,,sex
+#QUIC,,
+#TCP/UDP,,
+#BGP,,
+#Routing,,
+#IP,,
+#IPv4,,
+#IPv6,,
+#SNI,,
+#HTTP/0.9,,
+#HTTP0.9,,
+#HTTP2,,
+#HTTP/2,,
+#HTTP3,,
+#HTTP/3,,
+#TLS,,
+#TCP,,
+#UDP,,
```

Comment:
I think we can also the retweets ids in the same file

Reply:
Yeah im testing out that way